sdk/serialization/azure-xml/add-shaded-code-comment.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Python version 3.4 or higher is required to run this script.
+
+# Simple helper script that adds a single-line (//) Java comment to shaded code
```

Comment:
could it be done by spotless apply?

Reply:
Tried using Spotless, it can only fully replace the whole license header, which will strip out the existing license header instead of prepending to it.